Zstandard.Net/ZstandardStream.cs

```diff
@@ -156,6 +156,7 @@ protected override void Dispose(bool disposing)
 
             if (this.isDisposed == false)
             {
+                if (!this.isClosed) ReleaseResources(flushStream: false);
```

Comment:
If I understand correctly, this means that if a compressing `ZstandardStream` is used in a conventional `using` block, then after the block completes and implicitly `Dispose` has been called - that the data written into the `ZstandardStream` instance is not necessarily written to the underlying stream?

Reply:
This is the same behavior prior to my change. What I added was the ReleaseResources( ), which at least frees the memory stopping the leak.